src/cookbook/gradual-composition-api-migration.md

```diff
@@ -0,0 +1,245 @@
+# Gradual Composition API Migration
+
+The [Composition API](/guide/composition-api-introduction.html) is an advanced feature introduced in Vue 3. It is purely additive, and the Options API is not legacy. You may find, however, that the Composition API can be a useful feature for more flexible large-scale architectures. So how would we go about introducing it in a larger system, gradually? What follows is merely a recommendation, there are many potential ways forward.
```

Comment:
We can mention it, sure- same response here as to some of the other questions before- we're using Vue 2 with a Vue 3 package because the purpose of the article is to show how people can migrate an existing Vue 2 project, not a greenfield Vue 3 one. It's also mentioned that this isn't the only way to go about it.